custom_components/myskoda/climate.py

```diff
@@ -123,10 +130,116 @@ async def async_set_temperature(self, **kwargs):  # noqa: D102
         await self.coordinator.myskoda.set_target_temperature(
             self.vehicle.info.vin, temp
         )
-        _LOGGER.info("AC disabled.")
+        _LOGGER.info("Target temperature for AC set.")
 
     def required_capabilities(self) -> list[CapabilityId]:
         return [
             CapabilityId.AIR_CONDITIONING,
             CapabilityId.AIR_CONDITIONING_SAVE_AND_ACTIVATE,
         ]
+
+
+class AuxiliaryHeater(MySkodaEntity, ClimateEntity):
+    """Auxiliary heater control for MySkoda vehicles."""
+
+    entity_description = ClimateEntityDescription(
+        key="auxiliary",
+        translation_key="auxiliary",
```

Comment:
Please rename this to `auxiliary_heater` to be more exact as to what it is about.